hadoop-ozone/integration-test/src/test/java/org/apache/hadoop/ozone/StandardOutputTestUtil.java

```diff
@@ -28,7 +28,7 @@
 /**
  * Utility class to check standard output.
  */
-public class TestStandardOutputUtil {
+public class StandardOutputTestUtil {
```

Comment:
Thanks @JyotinderSingh for the patch.

Nit: I think `StandardOutputTestBase` would better indicate that this class is to be extended, rather than a collection of utility methods.  (I know `Util` was part of the original name, too.)

Reply:
I agree, using Base instead of Util makes it more obvious as to how this class is supposed to be used. Thank you for the suggestion, I have made this change.